src/main/java/com/davidthomasbernal/stardict/parsers/SynParser.java

```diff
@@ -14,7 +14,7 @@
     private final DictionaryInfo dictionaryInfo;
     private final DictionaryIndex dictionaryIndex;
     private Logger logger = Logger.getLogger(this.getClass().getName());
-    private boolean tolerateInfoMismatch = false;
```

Comment:
For clarity: Let's keep this initialization. (false may be the default value, but folks like me don't remember.)

Reply:
it's easy to remember these defaults: 
boolean fields: false, number fields: 0  objects: null. In contrast with C/C++ you can always be sure in this values. 
In fact for those who remember this it takes additional attention to check the init value. My IDE shows this default value with gray color and shows an intention action to fix that

Reply:
"In contrast with C/C++ you can always be sure in this values." - oh ok.